main_test.go

```diff
@@ -91,4 +91,117 @@ func TestFilterVariants(t *testing.T) {
 
 		require.Equal(t, expectedVariants, filtered)
 	}
+
+	t.Log("exact match for module and multiple variants")
+	{
+		filtered, err := filterVariants("module1", `variant1\nvariant2`, variants)
+		require.NoError(t, err)
+
+		expectedVariants := gradle.Variants{
+			"module1": []string{"variant1", "variant2"},
+		}
+
+		require.Equal(t, expectedVariants, filtered)
+	}
+
+	t.Log("exact match for module and multiple variants")
```

Comment:
I may miss something but this seems to be == with the one above

Reply:
You are right! The curse of copy-paste. 😄